php-malware-finder/whitelists/guzzle.yar

```diff
@@ -6,6 +6,9 @@ whitelist Guzzle.
 
 When a new version of Guzzle is released, you generate hashes for newly-violating files with something like:
 yara -r ./php.yar /path/to/guzzle-NEWVERSION | sed -e 's/.* //' | xargs sha1sum | uniq
+
+BEFORE GENERATING NEW HASHES, be sure to run composer i --no-dev in the guzzle directory, because guzzle
+has sub-dependencies (like psr7) which introduce their own (php://temp) false positives otherwise!
```

Comment:
I've added this comment to help Future Dan (or whoever) to not make the same mistake as I did in PR #4!